### PR TITLE
Update analytics-tools.mdx

### PR DESCRIPTION
**Pull request by i-shivamsoni**
**Description**

Updated the list of OP Dashboards & Added additional tools/resources as mentioned in issue https://github.com/ethereum-optimism/docs/issues/200

**Metadata**

- Fixes #[https://github.com/ethereum-optimism/docs/issues/200]

### CODE REVIEW

File: pages/builders/tools/analytics-tools.mdx [hunk 1]
```diff
@@ -30,3 +30,17 @@ Here are some of our favorite dashboards so far:
...
+*   [OP Mainnet Protocol Metrics](https://dune.com/optimismfnd/Optimism)
+*   [Popular Apps on OP Mainnet and Project Usage Trends](https://dune.com/optimismfnd/Optimism-Project-Usage-Trends)
+*   [OP Token House Delegates](https://dune.com/optimismfnd/optimism-op-token-house)
+*   [Superchain NFTs](https://dune.com/oplabspbc/superchain-nfts)
+
+## Additional Tools and Resources
+
+Here are some additional tools and resources for OP Mainnet analytics and development:
+
+* DeFi Stats: [Optimism's DeFiLlama Page](https://defillama.com/chain/Optimism/)
```
Comment (smartcontracts) [inline pages/builders/tools/analytics-tools.mdx:43]:
```suggestion
* DeFi Stats: [OP Mainnet's DeFiLlama Page](https://defillama.com/chain/Optimism/)
```

**Review by smartcontracts — COMMENTED** [review]
Looking good! I think you will need to run `pnpm fix` to fix linting issues but otherwise seems good to me

File: pages/builders/tools/analytics-tools.mdx (hunk 1 above)
Comment (i-shivamsoni) [reply pages/builders/tools/analytics-tools.mdx]: Thank you for the quick review, I have made the changes you suggested.

**Review by smartcontracts — APPROVED** [review]

**Comment by smartcontracts** [comment]
@i-shivamsoni could you add "Numba" and "Dework" to the words.txt file? Otherwise spellcheck will fail. Run `pnpm spellcheck` to see this for yourself.

**Review by cpengilly — APPROVED** [review]
@i-shivamsoni thank you for your contribution! 🎉

**Pull request merged**